pocs/serial_handlers/protocol_arduinosimulator.py

```diff
@@ -251,9 +251,15 @@ def read(self, size=1):
             if timeout_obj.expired():
                 break
         response = bytes(response)
-        self.logger.info('FakeArduinoSerialHandler.read({}) -> {!r}', size, response)
+        # if size > 1:
+        #     self.logger.debug('FakeArduinoSerialHandler.read({}) -> {!r}', size, response)
```

Comment:
?

Reply:
Removing commented out code.

It was too noisy because size=1 most of the time; this is because it gets called by readline in io.IOBase, the base of the serial impls. I experimented with only printing when size>1, but settled on adding a readline impl that prints the line that is read.